### ISSUE TITLE
Banca d'Italia Currency Converter Extension Bug

### ISSUE THREAD

**Issue by CheatGZ** [raycast/extensions#1050]
# Extension – [Banca d'Italia Currency Converter](https://github.com/raycast/extensions/tree/main/extensions/banca-d-italia-currency-converter)

Author:@FezVrasta

## Description

Unable to use the "Banca d'Italia Currency Converter" extension, the current problem is reporting an error, the previous problem encountered was that some currency conversions were not working properly.

## Steps To Reproduce

1. Open extension

1. Select currency

## The current behavior

Unable to convert currency

Report an error:There are no rates for the requested date

<img width="786" alt="iShot2022-03-09 18 18 40" src="https://user-images.githubusercontent.com/36128970/157422031-db364b9a-6144-420f-a19e-2209e834e2ab.png">

<img width="786" alt="iShot2022-03-09 18 22 19" src="https://user-images.githubusercontent.com/36128970/157422456-e81d126c-0910-4769-898d-49b1a8fe8f5d.png">

## The expected behavior

Can select the currency normally and convert it

### Raycast version

Version:1.29.1

**Comment by FezVrasta** [comment]
What's the expected behavior? The extension is telling you there are no available conversion rates for the date you selected.

**Comment by CheatGZ** [comment]
Maybe I misunderstood this extension, I expected this extension to help me convert currencies, but now there is no way to do it.

When I used it today, I tried all the Reference Date, but all of them prompted an error, which prevented me from using it for currency conversion.

Yesterday also I tried to use it and no error was reported, but I couldn't convert between certain currencies properly, I mainly use CNY.

Am I using the extension in the wrong way? I'm currently in China and using a VPN to access the internet.


<details>
<summary>Record Screen</summary>

https://user-images.githubusercontent.com/36128970/157427940-e7adf943-d1f3-4a60-9bd4-b732046114da.mp4

</details>
